awx_collection/plugins/modules/tower_group.py

```diff
@@ -41,49 +46,6 @@
       description:
         - Variables to use for the group, use C(@) for a file.
       type: str
-    credential:
-      description:
-        - Credential to use for the group.
-      type: str
-    source:
-      description:
-        - The source to use for this group.
-      choices: ["manual", "file", "ec2", "rax", "vmware", "gce", "azure", "azure_rm", "openstack", "satellite6" , "cloudforms", "custom"]
-      type: str
-    source_regions:
-      description:
-        - Regions for cloud provider.
-      type: str
-    source_vars:
-      description:
-        - Override variables from source with variables from this field.
-      type: str
-    instance_filters:
-      description:
-        - Comma-separated list of filter expressions for matching hosts.
-      type: str
-    group_by:
-      description:
-        - Limit groups automatically created from inventory source.
-      type: str
-    source_script:
-      description:
-        - Inventory script to be used when group type is C(custom).
-      type: str
-    overwrite:
-      description:
-        - Delete child groups and hosts not found in source.
-      type: bool
-      default: 'no'
-    overwrite_vars:
-      description:
-        - Override vars in child groups and hosts with those from external source.
-      type: bool
-    update_on_launch:
-      description:
-        - Refresh inventory data from its source each time a job is run.
-      type: bool
-      default: 'no'
```

Comment:
We should mark this as a backward-incompatible change, in the README.md we're just making a rolling list.

Reply:
Done